buoy.php

```diff
@@ -107,13 +107,24 @@ public static function registerL10n () {
      * @return void
      */
     public static function initialize () {
-        // Make sure the WP REST API plugin is installed.
-        // We can remove this if block once that plugin is merged
-        // into WP Core.
-        require_once ABSPATH.'wp-admin/includes/plugin.php';
-        if (is_plugin_inactive('rest-api/plugin.php')) {
-            if (is_wp_error(activate_plugin('rest-api/plugin.php'))) {
-                self::install_plugin_dependency('rest-api');
+        // WordPress versions prior to 4.7 did not fully merge the
+        // WP REST API, on which we (partially) rely. In version 4.7,
+        // however, the REST API's important parts for our use case,
+        // the so-called "Content API" did indeed land in core. This
+        // means we no longer need to force-install it ourselves.
+        //
+        // See https://codex.wordpress.org/Version_4.7 highlights.
+        //
+        // For now, make sure the WP REST API plugin is installed and
+        // activated for users still running older WordPress versions.
+        // We can remove this check once enough Buoys update to v4.7.
+        global $wp_version;
+        if (version_compare('4.7', $wp_version) === 1) {
```

Comment:
These are equivalent: 
`version_compare($wp_version, '4.7') < 0`
`version_compare('4.7', $wp_version) === 1`

and `version_compare($wp_version, '4.7') >= 0` is the inverse of what we want because I got confused.